modules/wayprompt/hm.nix

```
@@ -0,0 +1,33 @@
+{ config, lib, ... }:
+let
+  c = color: "0x${color}";
+  cfg = config.stylix.targets.wayprompt;
+in
+{
+  options.stylix.targets.wayprompt.enable = config.lib.stylix.mkEnableTarget "wayprompt" true;
+
+  config = lib.mkIf (config.stylix.enable && cfg.enable) {
+    programs.wayprompt.settings.colours = with config.lib.stylix.colors; {
+      background = c base00;
```

both work, but I personally prefer this:
```suggestion
  cfg = config.stylix.targets.wayprompt;
in
{
  options.stylix.targets.wayprompt.enable = config.lib.stylix.mkEnableTarget "wayprompt" true;

  config = lib.mkIf (config.stylix.enable && cfg.enable) {
    programs.wayprompt.settings.colours = lib.mapAttrs (_: color: "0x${color}") 
      with config.lib.stylix.colors; {
        background = base00;
```

---

sorry, didn't noticed your suggestion before format fix commit, i like it